source/javascripts/controllers/_MainController.js.erb

```diff
@@ -31,6 +31,7 @@ import _ from "underscore";
 			$scope.requestService = requestService;
 			$scope.osService = osService;
 
+			viewModel.showUpdateYmlInRepositoryModal = false;
```

Comment:
Just some nitpicking - call it like `viewModel.isUpdateYmlInRepositoryModalVisible`, so it indicates it's a boolean flag, not an action to call.